wikimedia/wikipedia-ios
# Add empty state for no other article languages

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T277066

### Notes
* This is hooking into our existing empty state helper methods to add an empty view when there are no other article languages to choose from.

### Test Steps
1. Go to article for "Myriam Gurba" on EN Wikipedia.
2. Tap the languages button
3. Confirm you see empty state as described in the Phab ticket (note I didn't make any changes to the navigation bar or search to match mocks exactly, see [Phab comment](https://phabricator.wikimedia.org/T277066#7039973) for approval). Confirm it works across device and type sizes, orientations, and themes.
4. Regression - Go to an article with languages (any common one like Cat). Confirm languages show and work as before.
5. Regression - Go to languages list page from Settings and Search, confirm they work as before.

### Screenshots/Videos *(if applicable)*
![Screen Shot 2021-05-03 at 5 51 48 PM](https://user-images.githubusercontent.com/3620196/116942985-49b9ee00-ac38-11eb-80d7-3d2c861bdc72.png)
![Screen Shot 2021-05-03 at 5 52 00 PM](https://user-images.githubusercontent.com/3620196/116942994-4de60b80-ac38-11eb-890c-9dfaa44c27df.png)
